includes/shortcodes.php

```diff
@@ -74,6 +74,10 @@ function give_donation_history( $atts ) {
 		ob_start();
 		give_get_template_part( 'history', 'donations' );
 
+		if ( ! empty( $content ) ) {
+			echo do_shortcode( $content );
+		}
+
 		return ob_get_clean();
```

Comment:
@raftaar1191 Add a filter here which help developer to change the position of the shortcode.

```
return apply_filters( 'give_donation_history_shortcode_html', ob_get_clean(), $content,$atts );
```